src/table_printer/table_printer.hpp

```diff
@@ -18,6 +18,12 @@
 #include <util/code_optimization.hpp>
 #include <table_printer/table_element_printers.hpp>
 
+#ifdef __APPLE__
+#include <os/log.h>
+#undef MIN
+#undef MAX
```

Comment:
It makes me nervous when I see this kind of preprocessor fiddling in header files. Do we really need to import <os/log.h> in the header file, or can we just import it into table_printer.cpp?

Reply:
Good call; I'll try to move it to the .cpp file.